fuzz/.gitignore

```diff
@@ -0,0 +1,4 @@
+target
+corpus
+artifacts
+coverage
```

Comment:
this entire `fuzz` crate should not be here.

Also update the `.gitignore` in `grandine` (not this one thus), to include `.DS_Store`.